Representable/representable
# Privacy and Tracking Consent Banner

* Added a privacy and tracking consent banner.
* Updated Privacy policy to reflect the third-party tracking services we use.
* Added an account settings pane in the dashboard (can be changed in the future, this is the closest we've got to an account page).

<img width="1552" alt="Screen Shot 2020-07-13 at 17 51 12" src="https://user-images.githubusercontent.com/8714338/87357228-8bef3300-c531-11ea-9df5-57b6a911854b.png">

**How does tracking consent work?**

- Mixpanel is now disabled by default. Google Analytics will be auto-enabled, so we can track unique page visits.
- Users will be presented with a banner that nudges them to opt-in or opt-out from tracking on their first visit. If they select an option, the banner will not show again and the state will be saved in the browser's `localStorage`.
- Users can update their preferences by pressing on "Tracking Settings" in the footer or on their dashboard page.
- This PR only implements a client-side settings model. This means that if the same user visits the website from different browsers, they will have to Opt-in/Opt-out by again.

**Why client-side?**
This is the most basic and common way of saving state about our users. In the future, we might also need to implement a server-side account setting for logged in users, but that's a lot of work for now, and the payoff is minimal (logged in users will be the only ones to benefit).

**To test this PR:**
1. Open an incognito window.
2. Open the browser's console.
3. Type `localStorage.getItem("user-consent");` Return should be null.
4. Approve tracking in the banner.
5. Type `localStorage.getItem("user-consent");` in the console. Return should be true.
6. Go to footer. Press on "Tracking Settings". Opt out of tracking.
7.  Type `localStorage.getItem("user-consent");` in the console. Return should be false.
8. Log in. Go to the account dashboard.
9. Check that the state is "opted out".
10. Toggle the state a few times and check the console.

**Additions:**

**Privacy and Tracking Notice**
<img width="1552" alt="Screen Shot 2020-07-13 at 17 51 12" src="https://user-images.githubusercontent.com/8714338/87357244-94476e00-c531-11ea-84e4-a613e5fb6a6f.png">
**"Tracking Settings" footer menu option. On click, it shows the banner again.**
<img width="949" alt="Screen Shot 2020-07-13 at 17 51 18" src="https://user-images.githubusercontent.com/8714338/87357255-98738b80-c531-11ea-8e51-6f7ca1404a2b.png">

**"Account Settings" dashboard pane. It includes a "Reset Password" button and a menu that displays the state of your tracking settings. This can be expanded in the future to accommodate server-side tracking.**

- Opt-in state.
<img width="929" alt="Screen Shot 2020-07-13 at 17 51 28" src="https://user-images.githubusercontent.com/8714338/87357270-a2958a00-c531-11ea-8fde-ed290ebed1c3.png">
- Opt-out state.
<img width="948" alt="Screen Shot 2020-07-13 at 17 51 32" src="https://user-images.githubusercontent.com/8714338/87357279-a75a3e00-c531-11ea-9623-d578df6037cb.png">

**Privacy Policy** The privacy policy now mentions Mixpanel and Google Analytics.

<img width="932" alt="Screen Shot 2020-07-13 at 17 56 20" src="https://user-images.githubusercontent.com/8714338/87357544-2cddee00-c532-11ea-8439-4a391ef13f86.png">
